.gitattributes

```diff
@@ -0,0 +1,11 @@
+docs/* linguist-documentation
+
+examples/* -linguist-detectable
+tutorial/* -linguist-detectable
```

Comment:
Does it make sene to move `examples/` and `tutorial` _into_ the `docs/` directory?